beastify/popup/choose_beast.html

```diff
@@ -7,9 +7,11 @@
   </head>
 
 <body>
-  <div class="beast">Frog</div>
-  <div class="beast">Turtle</div>
-  <div class="beast">Snake</div>
+  <div class="button beast">Frog</div>
+  <div class="button beast">Turtle</div>
+  <div class="button beast">Snake</div>
+  <div class="button clear inactive">Reload</div>
+  <div class="button clear inactive">Blank</div>
```

Comment:
I don't think we should have 2 buttons. We would only need one, that restores the page to its pre-beastified state. "Reset" might be a better name than "reload".